src/models/network.ts

```diff
@@ -39,8 +40,9 @@ const mainnet = Networks.add({
   pubkeyhash: versionBytes['mainnet']['p2pkh'],
   privatekey: 0x80,
   scripthash: versionBytes['mainnet']['p2sh'],
+  bech32prefix: 'ht',
```

Comment:
What is it for? We don't have bech32 addresses in Hathor.

Reply:
We don't, but since the new version of bitcore has this new parameter i thought we should include it.